databricks/koalas/tests/test_indexes.py

```diff
@@ -1583,46 +1583,76 @@ def test_intersection(self):
         # other = MultiIndex
         pmidx = pd.MultiIndex.from_tuples([("a", "x"), ("b", "y"), ("c", "z")])
         kmidx = ks.from_pandas(pmidx)
-        self.assert_eq(
-            pidx.intersection(pmidx), kidx.intersection(kmidx).sort_values(), almost=True
-        )
-        self.assert_eq(
-            (pidx + 1).intersection(pmidx),
-            (kidx + 1).intersection(kmidx).sort_values(),
-            almost=True,
-        )
+        if LooseVersion(pd.__version__) < LooseVersion("1.2.0"):
+            self.assert_eq(
+                kidx.intersection(kmidx).sort_values(),
+                kidx._kdf.head(0).index.rename(None),
```

Comment:
The difference must be only its name, right?
Then we should use `pidx` and `pmidx` for the expected values? e.g.,

```py
pidx.intersection(pmidx).rename(None)
```

Reply:
Sure, thanks! Looks more clearer.

Reply:
@ueshin Oh, no this is because the expected result is `Int64Index`, but the actual result of pandas < 1.2.0 is `MultiIndex`.

Reply:
Got it.